app/ui-react/syndesis/src/modules/integrations/locales/integrations-translations.en.json

```diff
@@ -91,15 +91,21 @@
       "saveAndPublish": "Save and publish"
     },
     "choiceForm": {
-      "conditionDescription": "Provide a condition that you want to evaluate (for example, ${in.header.type} == 'note' or ${in.body.title} contains 'Important').",
+      "conditionDescription": "Provide a condition that you want to evaluate (for example, ${header.type} == 'note' or ${body.title} contains 'Important').",
       "conditionName": "Condition",
-      "conditionPlaceholder": "Simple language expression",
+      "conditionPlaceholder": "Condition expression",
       "addCondition": "Add another condition",
       "addConditionTitle": "When",
       "forAllIncomingData": "For all incoming data",
-      "otherwise": "If incoming data does not match all of the above conditions",
+      "otherwise": "When none of the above conditions matches to the incoming data",
```

Comment:
Typo, I think you want `When none of the above conditions match the incoming data`

Reply:
👍 fixed